lib/resources/r4/encounter.yaml

```diff
@@ -527,6 +527,31 @@ fields:
         "reference": "Organization/619848"
       }
     }
+  note: >
+    <ul>
+      <li>serviceProvider cannot be provided when Encounter.location.location is present.</li>
+    </ul>
+
+- name: location
+  required: 'Yes'
+  type: Reference (Location)
+  description: The location where the encounter takes place.
+  action: create
+  example: |
+    {
+      "location": [
+        {
+          "location": {
+            "reference": "Location/21250409"
+          }
+        }
+      ]
+    }
+  note: >
+    <ul>
+      <li>Only one location reference is permitted.</li>
+      <li>location cannot be provided when Encounter.serviceProvider is present.</li>
```

Comment:
nit: location in ```code```.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/803/commits/62e556c0c21d59fccf0d1ab62663a549c0045d74#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR552